release-notes/README.md

```diff
@@ -4,7 +4,36 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
-## March 3, 2021
```

Comment:
We should keep March 3 for the Android Analytics and add a new section for the Xamarin release on March 4. 
Also, could we use a similar format as we have for Unity/Flutter releases: https://aep-sdks.gitbook.io/docs/release-notes#acp-unity-packages

```
## March 4, 2022
### ACP Xamarin frameworks
Updated the Android and iOS native libraries for the following Xamarin frameworks:
* Android Analytics framework v1.0.1 updated to v1.2.10 (Android) and v2.5.1 (iOS)
* ...